Sources/NIOHTTP2/HTTP2StreamChannel.swift

```diff
@@ -615,7 +621,7 @@ internal extension HTTP2StreamChannel {
     ///
     /// - parameters:
     ///     - frame: The `HTTP2Frame` received from the network.
-    func receiveInboundFrame(_ frame: HTTP2Frame) {
+    func receiveInbound(_ message: Message) {
```

Comment:
I think I'd rather that this didn't change, and instead that we added a line to convert the frame to `Message` in this method.